.php-cs-fixer.dist.php

```diff
@@ -23,6 +23,7 @@
 EOF;
 
 $finder = PhpCsFixer\Finder::create()
+    ->ignoreVCSIgnored(true)
```

Comment:
maybe we should request Symfony to support nested `.gitignore` files?

Reply:
I might try to submit the PR myself... If I find some spare time for that :)

Reply:
See https://github.com/symfony/symfony/pull/43150.